Clean Swift/Unit Tests (Quick-Nimble).xctemplate/___FILEBASENAME___WorkerSpec.swift

```diff
@@ -12,25 +12,24 @@ import Nimble
 
 class ___VARIABLE_sceneName___WorkerSpec: QuickSpec {
     override func spec() {
-        
+
         // MARK: - Subject Under Test (SUT)
-        
+
         var sut: ___VARIABLE_sceneName___Worker!
-        
+
         // MARK: - Test Doubles
-        
-        
+
         // MARK: - Tests
-        
+
         beforeEach {
             setupWorker()
         }
-        
+
         afterEach {
             sut = nil
         }
-        
-        // MARK: Use Cases
+
+        // MARK: - Use Cases
```

Comment:
```suggestion
        // MARK: - Methods
```

Can we change this to ‘methods’ since the `func`s may go beyond the use cases?

Reply:
OK